f2/src/forms/ContactInfoForm.js

```diff
@@ -20,7 +20,7 @@ const validate = (values) => {
   //condition for an error to occur: append a lingui id to the list of error
   if (
     values.email !== '' &&
-    !new RegExp(/^\w+@[a-zA-Z_]+?\.[a-zA-Z]{2,3}$/).test(values.email)
+    !new RegExp(/^\w+@[a-zA-Z_-]+?\.[a-zA-Z]{2,3}$/).test(values.email)
```

Comment:
I'm still concerned that we might mark legitimate emails as errors (for example, where the first part of the email contains a "+"). I'm thinking we should either relax it entirely and accept anything that has a "@" in it, or have a more complicated regex such as in https://www.w3resource.com/javascript/form/email-validation.php.

thoughts? @ngosset @davidheerema any thoughts?